Add Initial Salary Survey Results (#1184)
writethedocs/www: docs/surveys/salary-survey/2019.rst

   Figure: Actual Role Breakdown

--------------

.. h.iha544grkavd:

Work Location
-------------

56% of respondents work on site at their employer’s office. 17% work
completely remotely, and the remaining 27% work partially remotely and
partially onsite.

Of the respondents that work completely on site, 55% do so by choice,
while for 45% it’s a requirement.

Of those that work entirely remotely, 74% do so by choice, while only
26% have no on site alternative available to them.

.. figure:: images/8.png
   :alt: Figure: Work Location

   Figure: Work Location

--------------

.. h.me9iqb10zfgx:

Section 2: Salary Information
=============================

This section contained questions that addressed the all-important salary
figure, additional benefits, level of satisfaction and reasons for
dissatisfaction.

.. h.7208juau05as:

Annual Salary
-------------

Note: as 97% (632) of respondents reporting working between 30 and 60
hours per week - a “full time” role - the 3% reporting fewer than 30
hours have been omitted from the figures in this section.

The median salary across all regions was $74,500 (meaning half of the
respondents earned more, and half earned less).

> This figure is perhaps not very representative as it takes into account

Reviewer: ```suggestion
This figure is not very representative as it does not take into account
```

> the socio-economic situation of the countries of the very highest

Reviewer: ```suggestion
the socio-economic situation in the countries of the very highest
```

   |                 | Respondents     | Total           |                 |
   |                 | (working 30+    |                 |                 |
   |                 | hours)          |                 |                 |
   +=================+=================+=================+=================+
   | North America   | 386             | 61%             | $90,000         |
   +-----------------+-----------------+-----------------+-----------------+
   | - USA           | 344             | 54.4%           | $93,600         |
   +-----------------+-----------------+-----------------+-----------------+
   | - Canada        | 42              | 6.6%            | $60,273         |
   +-----------------+-----------------+-----------------+-----------------+
   | Europe          | 164             | 25.9%           | $48,700         |
   +-----------------+-----------------+-----------------+-----------------+
   | - EU Member     | 112             | 17.2%           | $46,104         |
   | States          |                 |                 |                 |
   +-----------------+-----------------+-----------------+-----------------+

>    | - UK \*         | 36              | 5.5%            | $60,269         |

Reviewer 2: Might be nice to rejig these as proper footnotes:

```
diff --git a/docs/surveys/salary-survey/2019.rst b/docs/surveys/salary-survey/2019.rst
index 3cf3b711..340fbe98 100644
--- a/docs/surveys/salary-survey/2019.rst
+++ b/docs/surveys/salary-survey/2019.rst
@@ -319,7 +319,7 @@ Salary by Region/Country
    | - EU Member     | 112             | 17.2%           | $46,104         |
    | States          |                 |                 |                 |
    +-----------------+-----------------+-----------------+-----------------+
-   | - UK \*         | 36              | 5.5%            | $60,269         |
+   | - UK [#f1]_     | 36              | 5.5%            | $60,269         |
    +-----------------+-----------------+-----------------+-----------------+
    | - Germany       | 24              | 3.7%            | $59,430         |
    +-----------------+-----------------+-----------------+-----------------+
@@ -340,8 +340,11 @@ Salary by Region/Country
    | Given           |                 |                 |                 |
    +-----------------+-----------------+-----------------+-----------------+
 
-\* including Scotland - no respondents were reported from Wales or
-Northern Ireland
+
+.. rubric:: Footnotes
+
+.. [#f1] including Scotland - no respondents were reported from Wales or
+ Northern Ireland
```

Author: Yea, it's just an initial HTML<->RST import. We might move forward w/ publishing HTML, so I wanted to wait to polish it until we see how it looks.

Reviewer 2: Yeah, that is why I didn't go ahead and fix it all.

The two largest age groups (26-35 year olds and 36-45 year olds)
combined formed 67.5% of the total respondents. Only 4.6% of respondents
fell into the youngest age group, and there were no respondents in the
66+ age bracket.  

.. figure:: images/12.png
   :alt: Figure: Age

   Figure: Age

--------------

.. h.1w9hfqazunpy:

Gender Identity
---------------

Note: 3 respondents skipped this question, and 1 provided a nonsensical
answer which was discarded.

61% of the respondents identified as women, 36% as men, and 3% as
non-binary or “other”.

.. figure:: images/13.png
   :alt: Figure: Gender Identity

   Figure: Gender Identity

--------------

.. h.6ascb2ek053u:

Education Level
---------------

Note: one respondent skipped this question

95% of respondents had completed a college or university degree or
higher. Those attending technical college numbered less than 3%, and
those who completed high school only (including those who did some
college but did not achieve a formal qualification) accounted for the
remaining fraction.

.. figure:: images/15.png
   :alt: Figure: Education Level

   Figure: Education Level

--------------

.. h.hpkr7a16phmf:

> Location

Reviewer: ```suggestion
Geographical Location
```

--------

> 55 respondents left this question blank or provided a non-quantifiable

Reviewer: ```suggestion
18 respondents left this question blank or provided a non-quantifiable
```

response.

Out of the 594 valid responses:

.. table::  Most Common Geographical Locations

   +---------------+---------------------------+
   | Location      | Percentage of Respondents |
   +===============+===========================+
   | United States | 58%                       |
   +---------------+---------------------------+

>    | Canada        | 43                        |

Reviewer: ```suggestion
   | Canada        | 7%                        |
```

that overall less than 30% of respondents work individually.

.. figure:: images/4.svg
   :alt: Figure: Type of Role

   Figure: Type of Role

--------------

.. h.2u53cvl6ll1e:

Length of time in current role
------------------------------

9% of respondents indicated they had worked in their current role for
less than one full year.

20% indicated one year, and a further 8% indicated more than one year
but less than 2 years.

2 years but less than 5 years made up 35%. 5 years but less than 10
years was another 19%.

Veterans of 10 or more years in their current role made up 9%. Of these,
9 individual respondents reported 30 years or more, with one respondent
reporting 40 years (the top value entered).

.. figure:: images/5.svg
   :alt: Figure: Time in Current Role

   Figure: Time in Current Role

--------------

.. h.dfv7pdw0gekn:

Proportion of role officially related to documentation
------------------------------------------------------

65% of respondents reported that documentation was the whole of their
official job description, and 30% reported that it was part of their
official job description. This overwhelming majority (95% of all

> responses) is not surprising, given the focus of the Write The Docs

Reviewer: ```suggestion
responses) is not surprising, given the focus of the Write the Docs
```

community.

> What was interesting, however, was the statistically relevant number of

Reviewer: ```suggestion
What was interesting was the statistically relevant number of
```

respondents choosing the other two options: 4% said that although
documentation was not part of their job description, they were expected
to perform documentation-related tasks anyway, and 1.5% said that
although documentation was not part of their job description, and they

> were not expected to perform documentation-related tasks, they did

Reviewer: ```suggestion
were not expected to perform documentation-related tasks, they did.
```

> anyway.

Reviewer: ```suggestion

```

.. figure:: images/8.svg
   :alt: Figure: Work Location

   Figure: Work Location

--------------

.. h.me9iqb10zfgx:

Section 2: Salary Information
=============================

This section contained questions that addressed the all-important salary
figure, additional benefits, level of satisfaction and reasons for
dissatisfaction.

.. h.7208juau05as:

Annual Salary
-------------

Note: as 97% (632) of respondents reporting working between 30 and 60
hours per week - a “full time” role - the 3% reporting fewer than 30
hours have been omitted from the figures in this section.

The median salary across all regions was $74,500 (meaning half of the
respondents earned more, and half earned less).

This figure is not very representative as it does not take into account
the socio-economic situation in the countries of the very highest
earners (out of the top 10 salary values, 9 were from the US and one
from the UK) and the very lowest (the bottom 10 salary values were from
 India, Asia, and Eastern Europe).

Figures grouped into regions make a more useful baseline from which to
determine what constitutes a “fair” salary.

.. h.mqgdllu8gaap:

> Salary by Region/Country

Reviewer: ```suggestion
Median Salary by Region and Country
```

.. h.e48bbqcfc9pg:

USA
~~~

The median salary for the entire United States was $93,600 - 25.6%
higher than the overall median salary.

Given that the US accounted for such a significant proportion of the
data, breaking the data down further, by the most represented states,
seems useful as well. Note that 12 of the respondents who indicated that
they lived in the USA did not provide a city or state, so those values
have been omitted from the following breakdown.

.. table::  Median Salary by US State

   +--------------+---------------+
   | State        | Median Salary |
   +==============+===============+
   | Washington   | $126,500      |
   +--------------+---------------+
   | California   | $120,000      |
   +--------------+---------------+
   | New York     | $105,000      |
   +--------------+---------------+
   | Pennsylvania | $96,350       |
   +--------------+---------------+
   | Texas        | $92,000       |
   +--------------+---------------+
   | Oregon       | $89,000       |
   +--------------+---------------+
   | Virginia     | $78,625       |
   +--------------+---------------+

.. figure:: images/map.svg

>    :alt: Figure: Median Salary by Region/Country

Reviewer: ```suggestion
   :alt: Figure: Median Salary by Region and Country
```

>    Figure: Median Salary by Region/Country

Reviewer: ```suggestion
   Figure: Median Salary by Region and Country
```

--------------

.. h.ldoc9nicw4s7:

Additional Benefits
-------------------

> Health insurance and vacation time were the most common benefits

Reviewer: ```suggestion
Health insurance and paid vacation time were the most common benefits
```

Institutions accounted for less than 2% of the respondents.

10 “Other” responses were entered, covering startups, government
contractors and independent units within larger organizations.

.. figure:: images/11.svg
   :alt: Figure: Type of Organization

   Figure: Type of Organization

--------------

.. h.k3uydo48dz7b:

Section 4: Respondent Demographics
==================================

Note: The questions in this section were optional.

.. h.fzwggzhwhbwh:

Age
---

Note: 3 respondents skipped this question

The two largest age groups (26-35 year olds and 36-45 year olds)
combined formed 67.5% of the total respondents. Only 4.6% of respondents
fell into the youngest age group, and there were no respondents in the
66+ age bracket.  

.. figure:: images/12.svg
   :alt: Figure: Age

   Figure: Age

--------------

.. h.1w9hfqazunpy:

Gender Identity
---------------

Note: 3 respondents skipped this question, and 1 provided a nonsensical
answer which was discarded.

61% of the respondents identified as women, 36% as men, and 3% as
non-binary or “other”.

.. figure:: images/13.svg
   :alt: Figure: Gender Identity

   Figure: Gender Identity

--------------

.. h.6ascb2ek053u:

> Education Level

Reviewer: ```suggestion
Highest Education Level Achieved
```

---------------

Note: one respondent skipped this question

95% of respondents had completed a college or university degree or

> higher. Those attending technical college numbered less than 3%, and

Reviewer: ```suggestion
higher. Those completing technical college numbered less than 3%, and
```

those who completed high school only (including those who did some
college but did not achieve a formal qualification) accounted for the
remaining fraction.

.. figure:: images/15.svg

>    :alt: Figure: Education Level

Reviewer: ```suggestion
   :alt: Figure: Highest Education Level Achieved
```

>    Figure: Education Level

Reviewer: ```suggestion
   Figure: Highest Education Level Achieved
```
